packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php

```diff
@@ -125,76 +131,77 @@
             <div class="header">
                 <div class="row">
                     <div class="col-12">
+                        @if (core()->getConfigData('sales.invoice_setttings.invoice_slip_design.logo'))
+                            <div class="image text-center">
+                                <img class="logo" src="{{ Storage::url(core()->getConfigData('sales.invoice_setttings.invoice_slip_design.logo')) }}" alt=""/>
+                            </div>
+                        @endif
                         <h1 class="text-center">{{ __('admin::app.sales.invoices.invoice') }}</h1>
                     </div>
                 </div>
+                <div class="row">
+                    <div class="col-12">
+                        <div class="col-6">
+                            <div class="merchant-details">
+                                <div>
+                                    <span class="merchant-details-title">{{ core()->getConfigData('sales.shipping.origin.store_name') ? core()->getConfigData('sales.shipping.origin.store_name') : '' }}</span>
```

Comment:
Change this as below

`core()->getConfigData('sales.shipping.origin.store_name') ?? '''`